english/call-a-doctor/hungry/bribe/bribe.md

```diff
@@ -0,0 +1,9 @@
+You reach for your sock and take out $20, offering it to the doctor.
+
+"I'll see what I can do," he says, leaving the room.
+
+You wait.
+
+And wait.
+
+[And wait.](../../marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.